htdocs/resource/list.php

```diff
@@ -237,34 +238,34 @@
 
 if ($ret)
 {
-    foreach ($object->lines as $resource)
+    foreach ($object->lines as $obj)
```

Comment:
$obj is reserved for result of database fetch.
For object, we prefer to use $object or if used $tmpobject or $nameofobject